tests/unit/test_checkpointing.py

```diff
@@ -254,8 +263,9 @@ def test_checkpoint_zero_optimizer(tmpdir, zero_stage):
             "enabled": True
         },
         "zero_optimization": {
-            "stage": zero_stage
-        },
+            "stage": zero_stage,
+            "zero_cpu_offload": use_cpu_offload
```

Comment:
I think the json config key is named `cpu_offload` above. Can we update them across the tests? This probably means the tests aren't being run with cpu offload turned on currently.